autogluon_text/README.md

```diff
@@ -0,0 +1,91 @@
+
+
+<div align="left">
+  <img src="https://user-images.githubusercontent.com/16392542/77208906-224aa500-6aba-11ea-96bd-e81806074030.png" width="350">
+</div>
+
+## AutoML Toolkit for Deep Learning
```

Comment:
Why have readme in autogluon_text? Why not in root?

Reply:
README.md is used as a long description in setup.py:
```
long_description = open('README.md').read()
```
Open for any suggestions what should we do here